src/app/clock-in-out-history/clock-in-out-history.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="flex-container">
+  <div class="history-container">
+    <ul>
+      <li
+        *ngFor="let element of this.clockInOutData; let odd = odd"
+        [ngClass]="{ 'odd-li': odd == true }"
```

Comment:
Luźna sugestia: 
Inny sposób na zapisanie tego (moim zdaniem bardziej czytelny) to:
[class.odd-li]="odd", czyli dodaj klasę odd-li jeśli pole odd zawiera wartości "falsy".